### ISSUE TITLE
[BUG] - Deployment on minikube fails: minio and postgresql charts not found

### ISSUE THREAD

**Issue by peytondmurray** [Quansight/qhub#1306]
### OS system and architecture in which you are running QHub

Linux 5.18.1-arch1-1 x86_64

### Expected behavior

Following the [minikube guide](https://github.com/Quansight/qhub/blob/main/docs/source/dev_guide/minikube.md), I expect to be able to deploy QHub locally.

### Actual behavior

Upon doing `python -m qhub deploy --config qhub-config.yaml --disable-prompt`, the deployment fails. All the preceeding steps work execute successfully.

It looks like minio 6.7.4 and postgresql 10.13.12 charts aren't present in https://charts.bitnami.com/bitnami.

### How to Reproduce the problem?

```bash
pip install qhub
minikube start --cpus 6 --memory 8192 --driver=docker
python qhub/tests/scripts/minikube-loadbalancer-ip.py
minikube addons enable metallb
mkdir qhub-local-deployment
cd qhub-local-deployment
mkdir data
cd data
qhub init local --project=localdeployment --domain github-actions.qhub.dev --auth-provider=password --terraform-state=local
python -m qhub deploy --config qhub-config.yaml --disable-prompt
```

### Command output

```bash session
...
[terraform]: ╷
[terraform]: │ Error: chart "minio" version "6.7.4" not found in https://charts.bitnami.com/bitnami repository
[terraform]: │ 
[terraform]: │   with module.kubernetes-conda-store-server.module.minio.helm_release.minio,
[terraform]: │   on modules/kubernetes/services/minio/main.tf line 7, in resource "helm_release" "minio":
[terraform]: │    7: resource "helm_release" "minio" {
[terraform]: │ 
[terraform]: ╵
[terraform]: ╷
[terraform]: │ Error: chart "postgresql" version "10.13.12" not found in https://charts.bitnami.com/bitnami repository
[terraform]: │ 
[terraform]: │   with module.kubernetes-conda-store-server.module.postgresql.helm_release.postgresql,
[terraform]: │   on modules/kubernetes/services/postgresql/main.tf line 7, in resource "helm_release" "postgresql":
[terraform]: │    7: resource "helm_release" "postgresql" {
[terraform]: │ 
[terraform]: ╵

Problem encountered: Terraform error
```

### Versions and dependencies used.

```bash
[~]$ kubectl version --output=yaml
clientVersion:
  buildDate: "2022-05-27T18:33:09Z"
  compiler: gc
  gitCommit: 3ddd0f45aa91e2f30c70734b175631bec5b5825a
  gitTreeState: archive
  gitVersion: v1.24.1
  goVersion: go1.18.2
  major: "1"
  minor: "24"
  platform: linux/amd64
kustomizeVersion: v4.5.4
serverVersion:
  buildDate: "2022-01-25T21:19:12Z"
  compiler: gc
  gitCommit: 816c97ab8cff8a1c72eccca1026f7820e93e0d25
  gitTreeState: clean
  gitVersion: v1.23.3
  goVersion: go1.17.6
  major: "1"
  minor: "23"
  platform: linux/amd64
```

### Compute environment

Tried with QHub 0.4.1 (from `pip install qhub`) as well as the latest commit (2687f13f54e3833234fc4282587be16521f0c84a).
minikube

**Comment by iameskild** [comment]
Hey @peytondmurray thanks for raising this! I will try to reproduce this error and report back 👍

**Comment by iameskild** [comment]
It looks like those versions of the minio and postgresql charts have been deprecated in some way as they are no longer hosted on https://charts.bitnami.com/bitnami (as mentioned above). 

A simple solution would be to pin these charts to a newer versions. That said there might have been a reason @costrouc pinned minio in the first place. Here is the comment right above the minio version: [`# last release that was Apache-2.0`](https://github.com/Quansight/qhub/blob/main/qhub/template/stages/07-kubernetes-services/modules/kubernetes/services/minio/main.tf#L13)

**Comment by peytondmurray** [comment]
Agreed, I saw that comment. Sounds like it might be best to wait for comment from @costrouc about supporting newer versions.

**Comment by iameskild** [comment]
This [bitnami issue](https://github.com/bitnami/charts/issues/10539) seems to suggest that a recent update that bitnami made to their `index.yaml` is causing trouble for many other projects as well.

I get a 404 error when I try to access their outdated `index.yaml`:
https://raw.githubusercontent.com/bitnami/charts/eb5f9a9513d987b519f0ecd732e7031241c50328/bitnami

**Comment by viniciusdc** [comment]
solved by #1308
